tests/dummy/app/pods/docs/components/phone-input/all-options/template.hbs

```diff
@@ -31,3 +31,16 @@
 
   {{demo.snippet "phone-input-separate-dial-option.hbs"}}
 {{/docs-demo}}
+
+<h2>`allowDropdown` option</h2>
+{{#docs-demo as |demo|}}
+  {{#demo.example name="phone-input-allow-dropdown-option.hbs"}}
+
+    <p>{{phone-input allowDropdown=false initialCountry="fr" update=(action "handleUpdate")}}</p>
```

Comment:
```suggestion
    <p>{{phone-input allowDropdown=false number=allowDropdownNumber initialCountry="fr" update=(action "updateAllowDropdown")}}</p>
```
If we add a number and a dedicated update method, it won't interfere with other phone-input occurences on this page.

Note that we probably want to update the `tests/dummy/app/pods/docs/components/phone-input/all-options/component.js` to implement the update method. That way, the sandbox UI will be 100% working.